string synth_name
string synth_text
bool force_resynth